src/main/webapp/app/faq/faq.routes.ts

```
@@ -0,0 +1,24 @@
+
+@Injectable({ providedIn: 'root' })
+export class FAQResolve implements Resolve<FAQ> {
+    constructor(private faqService: FAQService) {}
+
+    resolve(route: ActivatedRouteSnapshot): Observable<FAQ> {
+        const faqId = route.params['faqId'];
+        const courseId = route.params['courseId'];
+        if (faqId) {
+            return this.faqService.find(courseId, faqId).pipe(
+                filter((response: HttpResponse<FAQ>) => response.ok),
+                map((faq: HttpResponse<FAQ>) => faq.body!),
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)